src/nvim/message.c

```diff
@@ -126,29 +126,61 @@ int verb_msg(char_u *s)
   int n;
 
   verbose_enter();
-  n = msg_attr_keep(s, 0, FALSE);
+  n = msg_attr_keep(s, 0, false, false);
   verbose_leave();
 
   return n;
 }
 
-int msg_attr(const char *s, const int attr) FUNC_ATTR_NONNULL_ARG(1)
+int msg_attr(const char *s, const int attr)
+  FUNC_ATTR_NONNULL_ARG(1)
 {
-  return msg_attr_keep((char_u *)s, attr, false);
+  return msg_attr_keep((char_u *)s, attr, false, false);
 }
 
-int
-msg_attr_keep (
-    char_u *s,
-    int attr,
-    int keep                   /* TRUE: set keep_msg if it doesn't scroll */
-)
-  FUNC_ATTR_NONNULL_ARG(1)
+void msg_echo_show(const char *s, int attr)
```

Comment:
`msg_echo_show` is pretty confusing name, it is 3 synonyms to anyone without tribal knowledge. Maybe `msg_display` is an OK name, it is somewhat similar behavior as the existing `msg_puts_display`. (Other suggestions: `msg_fmt`, `msg_pretty`).